tfe/resource_tfe_registry_module_test.go

```diff
@@ -279,3 +625,111 @@ resource "tfe_registry_module" "foobar" {
  vcs_repo {}
 }`, rInt, token)
 }
+
+func testAccTFERegistryModule_privateRMWithoutRegistryName(rInt int) string {
+	return fmt.Sprintf(`
+resource "tfe_organization" "foobar" {
+ name  = "tst-terraform-%d"
+ email = "admin@company.com"
+}
+
+resource "tfe_registry_module" "foobar" {
+	organization    = tfe_organization.foobar.id
+  module_provider = "my_provider"
+  name            = "test_module"
+ }`,
+		rInt)
+}
+
+func testAccTFERegistryModule_privateRMWithRegistryName(rInt int) string {
+	return fmt.Sprintf(`
+resource "tfe_organization" "foobar" {
+ name  = "tst-terraform-%d"
+ email = "admin@company.com"
+}
+
+resource "tfe_registry_module" "foobar" {
+	organization    = tfe_organization.foobar.id
```

Comment:
these spaces are weird, they show up in my PR but in my VScode, they are all aligned. Not sure what's causing this. I will try running the formatter...any ideas?